src/main/java/reactor/netty/http/HttpInfos.java

```diff
@@ -65,7 +65,28 @@
 	 *
 	 * @return the decoded path portion from the {@link #uri()} without the leading and trailing '/' if present
 	 */
-	String path();
+	default String path() {
```

Comment:
thanks! prefer special casing here, vs trying to reverse engineer after the fact if there was a trailing slash or not in fullPath!